db/migration/migrations/000021_remove_mime_validation.up.sql

```diff
@@ -0,0 +1,9 @@
+BEGIN;
+
+CREATE OR REPLACE FUNCTION delete_data_in_healthit_products() RETURNS VOID as $$
+    BEGIN
+        DELETE FROM validations WHERE rule_name = 'generalMimeType';
+    END;
+$$ LANGUAGE plpgsql;
```

Comment:
@jdlane The migration wasn't working for me and I figured out why! You have to add the line
```
SELECT delete_data_in_healthit_products();
```
After the function definition so that it's actually run. I tried this locally and it worked, so if you could just add that here (and feel free to run it yourself and double check), that would be great!